src/logic/equiv/basic.lean

```diff
@@ -341,41 +341,29 @@ by { ext, simp }
 
 end perm_congr
 
+/-- Two empty types are equivalent. -/
+def empty_equiv_empty (α β : Sort*) [is_empty α] [is_empty β] : α ≃ β :=
```

Comment:
I think I'd expect this to be called `equiv_of_is_empty`

Reply:
I've changed the name accordingly.